### PR TITLE
STSMACOM-688 - Extend NotePopupModal

### PR DESCRIPTION
**Pull request by Terala-Priyanka**
## Purpose
Issue: [STSMACOM-688](https://issues.folio.org/browse/STSMACOM-688)

### CODE REVIEW

**Review by steveellis — APPROVED** [review]

**Comment by steveellis** [comment]
@Terala-Priyanka - Make sure to get an approval from Zak before merging. I looked at it and didn't see anything wrong but this is really his codebase.

**Comment by Terala-Priyanka** [comment]
@steveellis 
Thank you for your review. As this PR and associated tickets are linked/required by other PRs, and there are some comments. I am marking this PR as draft for now. I will request for re-review.

File: lib/Notes/NotePopupModal/NotePopupModal.js [hunk 1]
```diff
@@ -42,13 +44,14 @@ const propTypes = {
...
   popUpPropertyName: PropTypes.string.isRequired,
+
 };
 
 const defaultProps = {
   entityId: '',
   label: <FormattedMessage id="stripes-smart-components.notes.popupModal.label" />,
   closeLabel: <FormattedMessage id="stripes-smart-components.notes.close" />,
-  deleteLabel: <FormattedMessage id="stripes-smart-components.notes.popupModal.delete" />,
+  deleteLabel: <FormattedMessage id="stripes-smart-components.notes.popupModal.delete" />
```
Comment (NikitaSedyx) [inline lib/Notes/NotePopupModal/NotePopupModal.js:54]: trailing comma please

File: lib/Notes/NotePopupModal/NotePopupModal.js [hunk 2]
```diff
@@ -83,7 +86,7 @@ const NotePopupModal = ({
   popUpPropertyName,
   deleteLabel,
   closeLabel,
-  label,
+  label
```
Comment (NikitaSedyx) [inline lib/Notes/NotePopupModal/NotePopupModal.js:89]: same

**Review by JohnC-80 — APPROVED** [review]

File: lib/Notes/NotePopupModal/NotePopupModal.js (hunk 2 above)
Comment (Terala-Priyanka) [reply lib/Notes/NotePopupModal/NotePopupModal.js]: added trailing comma

File: lib/Notes/NotePopupModal/NotePopupModal.js (hunk 1 above)
Comment (Terala-Priyanka) [reply lib/Notes/NotePopupModal/NotePopupModal.js]: added trailing comma

**Pull request merged**